osu.Game/Input/Bindings/GlobalActionContainer.cs

```diff
@@ -148,6 +148,8 @@ public static IEnumerable<GlobalAction> GetGlobalActionsFor(GlobalActionCategory
             new KeyBinding(new[] { InputKey.Control, InputKey.Right }, GlobalAction.EditorSeekToNextHitObject),
             new KeyBinding(new[] { InputKey.Control, InputKey.Shift, InputKey.Left }, GlobalAction.EditorSeekToPreviousSamplePoint),
             new KeyBinding(new[] { InputKey.Control, InputKey.Shift, InputKey.Right }, GlobalAction.EditorSeekToNextSamplePoint),
+            new KeyBinding(new[] { InputKey.Control, InputKey.Up }, GlobalAction.EditorIncreasePlaybackSpeed),
+            new KeyBinding(new[] { InputKey.Control, InputKey.Down }, GlobalAction.EditorDecreasePlaybackSpeed),
```

Comment:
As per test failures, these key binds conflict with key bindings for nudging selected hit objects (ctrl-arrows).

Not sure I have a good idea how to fix this. Is one of the reasons why this wasn't immediately done when the issue was opened.

Reply:
> Does it only increase/decrease playback speed when there is no active selection?

It appears so yes.

Probably fine to replicate that behaviour.